lib/src/utils.dart

```diff
@@ -115,7 +115,8 @@ int? _lastNonWhitespace(String string, {bool excludeEscape = false}) {
       if (excludeEscape &&
           i != 0 &&
           i != string.length &&
```

Comment:
Can `i` ever be `string.length`?  I think this condition should actually be checking against `string.length - 1`. That said, this fix does appear to solve the crash as well

Reply:
Good point. That's actually a cleaner solution, so I'll change it to that. Thanks!